applications/GeoMechanicsApplication/custom_processes/apply_k0_procedure_process.cpp

```diff
@@ -22,16 +22,46 @@
 #include "utilities/math_utils.h"
 
 // Application includes
+#include "custom_constitutive/linear_elastic_law.h"
 #include "geo_mechanics_application_variables.h"
 #include "includes/kratos_flags.h"
-#include "includes/kratos_parameters.h"
+
+namespace
+{
+
+using namespace Kratos;
+
+void SetConsiderDiagonalEntriesOnlyAndNoShear(ModelPart::ElementsContainerType& rElements, bool Whether)
+{
+    block_for_each(rElements, [Whether](Element& rElement) {
+        auto pLinearElasticLaw =
+            dynamic_cast<GeoLinearElasticLaw*>(rElement.GetProperties().GetValue(CONSTITUTIVE_LAW).get());
+        if (pLinearElasticLaw) pLinearElasticLaw->SetConsiderDiagonalEntriesOnlyAndNoShear(Whether);
+    });
+}
+
+} // namespace
 
 namespace Kratos
 {
 
-ApplyK0ProcedureProcess::ApplyK0ProcedureProcess(ModelPart& model_part, const Parameters&)
-    : Process(Flags()), mrModelPart(model_part)
+ApplyK0ProcedureProcess::ApplyK0ProcedureProcess(ModelPart& model_part, const Parameters& rK0Settings)
+    : Process(Flags()), mrModelPart(model_part), mSettings(rK0Settings)
+{
+}
+
+void ApplyK0ProcedureProcess::ExecuteInitialize()
 {
+    if (UseStandardProcedure()) {
+        SetConsiderDiagonalEntriesOnlyAndNoShear(mrModelPart.Elements(), true);
+    }
+}
+
+void ApplyK0ProcedureProcess::ExecuteFinalize()
+{
+    if (UseStandardProcedure()) {
```

Comment:
Does clang format not take out the { } for this 1 line like statement?

Reply:
No, it does not remove the curly braces automatically. But it also does not put them back when I take them out manually. And so I did.